pkg/shell/active-pages-modal.jsx

```diff
@@ -25,12 +25,11 @@ import { Button } from "@patternfly/react-core/dist/esm/components/Button/index.
 import { Label } from "@patternfly/react-core/dist/esm/components/Label/index.js";
 import { Split, SplitItem } from "@patternfly/react-core/dist/esm/layouts/Split/index.js";
 import { Modal } from "@patternfly/react-core/dist/esm/components/Modal/index.js";
-import { useDialogs } from "dialogs.jsx";
```

Comment:
Commit message:

> Let's avoid that with the result.resolve() API — it's properly typed,
and TypeScript doesn't like it.

Supposedly that's meant to say "likes it"? :grin:

Reply:
I meant to say that it doesn't like passing `result.resolve()` directly to an `onClick` handler.  Will reword.